src/Translator.java

```diff
@@ -0,0 +1,16 @@
+import java.util.HashMap;
+
+public class Translator {
+    HashMap<Integer,String> numericAlpha = new HashMap<>();
```

Comment:
In het college van vanmiddag wordt het uitgelegd, maar volgens mij is het conventie om voor de variabel naam de superklasse van hashmap te zetten. Dit is volgens mij een Map.